svelte-cloudinary/src/lib/components/CldImage.svelte

```diff
@@ -82,6 +65,19 @@
 			console.warn(`Failed to preserve transformations: ${(e as Error).message}`);
 		}
 	}
-	let newSrc = getCldImageUrl({ ...imageProps, ...cldOptions });
 </script>
-<Image src={newSrc} {width} {height} {alt} {layout} {background} {priority} />
+
+<Image
+	{...imageProps}
+	cdn="cloudinary"
+	transformer={({ width, url, height}) => {
+		return getCldImageUrl({
+			...imageProps,
+			...cldOptions,
+			width,
+			height,
+			// @ts-ignore
+			src: url
```

Comment:
@matiasfha i also didnt know how to fix this TS error. `url` can be of type URL not only a string and `src` expects a string. the right fix would probably be me updating teht type in URL loader to also take a URL but for now not sure how to make this happy